data/trie/patriciaMerkleTrie.go

```diff
@@ -133,8 +133,10 @@ func (tr *patriciaMerkleTrie) Update(key, value []byte) error {
 		tr.root = newRoot
 		tr.oldHashes = append(tr.oldHashes, oldHashes...)
 
-		for _, hash := range oldHashes {
-			log.Trace("oldHashes after insert", "hash", hash)
+		if log.GetLevel() == logger.LogTrace {
+			for _, hash := range oldHashes {
```

Comment:
May take out into a function the whole section

Reply:
Done.